src/process.c

```diff
@@ -226,9 +226,7 @@ cc_oci_setup_shim (struct cc_oci_config *config,
 	ret = true;
 
 out:
-	if (tty_fd > 2) {
```

Comment:
This seems to have been done on purpose in commit f1c21527e9056c3f629baaa98849c67edd866cbb